src/Overlay.tsx

```diff
@@ -34,7 +34,7 @@ const widthMap = {
 /*TODO replace with shadow functional color variable when it's shipped to primer/primitives*/
 const StyledOverlay = styled.div<StyledOverlayProps & SystemCommonProps & SystemPositionProps & SxProp>`
   background-color: ${get('colors.bg.overlay')};
-  box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.12), 0px 8px 24px rgba(149, 157, 165, 0.2);
+  box-shadow: ${get('shadows.shadow.large')};
```

Comment:
It looks like the old `Dialog` uses `large` (https://github.com/primer/components/blob/main/src/Dialog.tsx#L22), but the old `SelectMenu` uses `small` (https://github.com/primer/components/blob/main/src/SelectMenu/SelectMenuModal.tsx#L37).  Might be worth checking with @vdepizzol to see which would would prefer in the short term